code/modules/urist/modules/weather.dm

```diff
@@ -0,0 +1,53 @@
+//weather effects
+
+/obj/effect/weather
+	name = "weather"
+	icon = 'icons/urist/weather.dmi'
+	icon_state = "bsnow"
+
+	layer = 5
+	anchored = 1 //prevents weather from being /draggable/
+	mouse_opacity = 0 //doesn't need to be clickable and is less of an annoyance for players
+
+/obj/effect/weather/blowingsnow
+	name = "blowing snow"
+	icon_state = "bsnow"
+	var/safe = 0 //safe == not arctic; ugly, but better than changing the vars on the map
+
+/obj/effect/weather/blowingsnow/Crossed(O as mob)
+	..()
+	if(!safe)
+		if(istype(O, /mob/living/))
+			var/mob/living/M = O
+			if(COLD_RESISTANCE in M.mutations)
```

Comment:
not necessarily under the scope of the PR, but something that will need to be done is to make this check for clothing with the appropriate temp resistance as well.